search.html

```diff
@@ -0,0 +1,163 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Mobility Web Component</title>
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <meta
+      name="viewport"
+      content="initial-scale=1,maximum-scale=1,user-scalable=no"
+    />
+    <script type="text/javascript">
+      if (/localhost/.test(window.location.hostname)) {
+        new EventSource("/esbuild").addEventListener("change", () => {
+          location.reload();
+        });
+      }
+    </script>
+    <script type="module" src="./index.js"></script>
+    <link rel="stylesheet" type="text/css" href="./output.css" />
+    <style>
+      ::-webkit-scrollbar {
+        width: 10px;
+      }
+      a {
+        text-decoration: underline;
+      }
+    </style>
+  </head>
+  <body class="p-8">
+    <div
+      id="doc"
+      style="display: none"
+      class="mx-auto h-96 max-w-3xl space-y-4"
+    >
+      <h1 class="text-3xl font-bold">Mobility Search Web Component</h1>
+      <p>This is a demo of the geOps Mobility Search Web Component.</p>
+      <p>This web component launches a search on the <b>geOps Stops API</b></p>
+      <p>Every parameters of the <b>geOps Stops API</b>:</p>
+      <ul class="pl-8">
+        <li class="list-disc">can be passed as a string attribute of the web component.</li>
+        <li class="list-disc">
+          can be passed as an URL parameter of this page, they will be
+          automatically apply to the web component.
+        </li>
+      </ul>
+      <p>
+        The list of parameters of the <b>geOps Stops API</b> can be found
+        <a href="https://developer.geops.io/apis/stops#parameters" target="_blank">here</a>.
+      </p>
+
+      <h2 class="text-xl font-bold">Usage example</h2>
+      <pre class="bg-slate-800 text-slate-200 p-4 rounded">
```

Comment:
Maybe we could add a hint like "Please view page source to get more details on how this demo is implemented". Otherwise someone could get the impression the whole demo is implemented only by the web component.

Reply:
we could add a link yes.